misk-web/web/packages/@misk/cli/src/utils/index.ts

```diff
@@ -32,25 +32,39 @@ export enum Files {
 
 export const JsonOptions = { spaces: 2 }
 
-export const cmdFromArgs = (exec: string) => {
-  const args = process.argv
-  args.splice(0, 1)
-  args[0] = exec
-  return args.join(" ")
-}
+export const prebuild = async (...args: any) =>
+  execute("miskweb prebuild", ...args)
 
-export const runCmd = (cmd: string) => {
-  try {
-    execSync(cmd, { stdio: "inherit", encoding: "utf-8" })
-  } catch (e) {
-    console.log(e.stdout)
-    console.log(e.stderr)
+export const execute = async (cmd: string, ...args: any) => {
+  const dir = args[0]
+  if (dir) cd(dir)
```

Comment:
pretty against single line conditionals, mind wrapping this in braces? 

https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/